include/common.h

```diff
@@ -14,9 +18,13 @@ void inspect_pointer(void *ptr)
 	bool tag = cheri_tag_get(ptr);
 
 	uint64_t offset = cheri_offset_get(ptr);
-	printf("Address: %04lx, Base: %04lx, End: %04lx Flags: %04lx, Length: %04lx, Offset: %04lx, "
-		   "Perms: %04lx, Type: %04lx, Tag: %d\n",
-		   address, base, base + length, flags, length, offset, perms, type, tag);
+
+	printf("*** CAPABILITY INFO for ***\n");
```

Comment:
"Capability info for" ... feels like half a sentence! Also, I don't think we need to shout at users :) "I think it would be fine to say something "Capability: %#lp\nTag: %d, ...". Put another way, I think the less extraneous text we add, the easier it is, in general, for users to pick out the important bits.

Reply:
Right, I've fixed this in the second commit (part of the PR) but, of course, the `for` is wrong. I thought at the beginning to add a parameter to the function giving the name of the variable we are giving info, e.g. `pointer_x`. However, this would have required much more refactoring and probably not that useful ATM.

Reply:
"Shouting" fixed in 3e87fb8.